src/mangosd/mangosd.conf.dist.in

```diff
@@ -42,7 +42,7 @@ ConfVersion=2015090801
 #	CharacterDatabaseConnections
 #		 Amount of connections to database which will be used for SELECT queries. Maximum 16 connections per database.
 #		 Please, note, for data consistency only one connection for each database is used for transactions and async SELECTs.
-#		 So formula to find out how many connections will be established: X = �_connections + 1
+#		 So formula to find out how many connections will be established: X = ą_connections + 1
```

Comment:
Is this symbol correct? - Someone telle me because then I can clean out some tabs from mangosd.conf.dist.in - ie this very line.